### PR TITLE
Use unique table name in testAccessControl

### PR DESCRIPTION
**Pull request by sopel39**

### CODE REVIEW

File: presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java [hunk 1]
```diff
@@ -4686,7 +4686,7 @@ public void testAccessControl()
     {
         assertAccessDenied("INSERT INTO orders SELECT * FROM orders", "Cannot insert into table .*.orders.*", privilege("orders", INSERT_TABLE));
         assertAccessDenied("DELETE FROM orders", "Cannot delete from table .*.orders.*", privilege("orders", DELETE_TABLE));
-        assertAccessDenied("CREATE TABLE foo AS SELECT * FROM orders", "Cannot create table .*.foo.*", privilege("foo", CREATE_TABLE));
+        assertAccessDenied("CREATE TABLE create_access_denied AS SELECT * FROM orders", "Cannot create table .*.create_access_denied.*", privilege("create_access_denied", CREATE_TABLE));
```
Comment (kokosing) [inline presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java]: the issue here is that this table is not actually created. So there is other table that was created and not removed. So you are fixing symptoms not a cause.

Comment (sopel39) [reply presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java]: There is another test: `testDescribeOutputNonSelect` which uses `foo` table too.

Comment (kokosing) [reply presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java]: That test also does not create a table.

Comment (sopel39) [reply presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java]: I think it does

**Pull request closed without merging**